src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2745,6 +2749,13 @@ namespace NServiceBus.Sagas
         public System.Collections.Generic.Dictionary<string, object> Properties { get; }
         public System.Type Type { get; }
     }
+    public class SagaIdGeneratorContext : NServiceBus.Extensibility.ContextBag
```

Comment:
Normally we have a context property on the context and do not inherit from the context bag so that you have to explicitly do context.Context.TryGet... I know it is a bit more verbose but it hides the extensibility part and doesn't expose it it "in your face style" in intellisense